src/test/java/com/wikia/webdriver/elements/mercury/pages/login/SignInPage.java

```diff
@@ -2,7 +2,7 @@
 
 import com.wikia.webdriver.common.core.Assertion;
 import com.wikia.webdriver.common.core.elemnt.Wait;
-import com.wikia.webdriver.elements.mercury.components.login.LoginArea;
+import com.wikia.webdriver.elements.mercury.components.loginAndSignup.LoginArea;
```

Comment:
Please split these two to login and ignup area:  
LoginArea goes to components.login.
RegisterArea goest to components.signup